various-receipes/friendly-structure-for-reporting/jest.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  verbose: true,
+  //testMatch: ['**/anti-pattern-flat-report.test.js'],

```

Comment:
why comment?

Reply:
@mikicho I don't want to run anti-patterns files